packages/core/src/pagination/register.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@cds/core/button/register.js';
+import '@cds/core/icon/register.js';
+import '@cds/core/input/register.js';
+import { registerElementSafely } from '@cds/core/internal';
+import { ClarityIcons } from '../icon/icon.service.js';
+import { angleIcon, stepForward2Icon } from '../icon/index.js';
```

Comment:
For internal icon imports load directly from the icon file. This helps with native esm loading performance not eagerly loading all icons in the index file. Also this helps treeshaking as each component is its own entrypoint/package. https://clarity.design/storybook/core/?path=/story/internal-documentation-using-icons--page 

```typescript
import { ClarityIcons } from '@cds/core/icon/icon.service.js';
import { angleIcon } from '@cds/core/icon/shapes/angle.js';
import { stepForward2Icon } from '@cds/core/icon/shapes/step-forward-2.js';
```